lib/src/screens/dashboard/desktop_widgets/desktop_wallet_selection_dropdown.dart

```diff
@@ -33,8 +35,8 @@ class _DesktopWalletSelectionDropDownState extends State<DesktopWalletSelectionD
   @override
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
-    return DropdownButton<WalletListItem>(
-      items: widget.walletListViewModel.wallets
+    return DropdownButton<DesktopDropdownItem>(
+      items: widget.walletListViewModel.dropdownItems
```

Comment:
Instead of making an abstract class to extend them both from it.
Better Create another new model with Icon and title and onSelected, and map them to this new model instead with their respective Icons (images for wallets, Add, and Restore Icons)